### PR TITLE
Update Reaque description

### PR DESCRIPTION
**Pull request by horaciocome1**
The text describing Reaque has changed from the last pull request, here made an update accordingly.
Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 ## <a name="R"> </a>R
 
-* [reaque](https://github.com/horaciocome1/reaque) - Repository for the Reaque android app. Reaque is an entire platform backed by [Firebase](https://firebase.google.com) that allows users to see and share experiences from Mozambique. Where you can find places, lyrics, poems and pictures from artists, non artists, and enthusiasts.
+* [reaque](https://github.com/horaciocome1/reaque) - Repository for the Reaque android app. Reaque is an mobile tool where we can share and find things related to Mozambique - the people and things there. From reflections and culture, throught history and gastronomy, to tourim and even fashion, if it is something about there, you should share and find it in Reaque.
```
Comment (thatfiredev) [inline README.md]:
```suggestion
* [reaque](https://github.com/horaciocome1/reaque) - Repository for the Reaque android app. Reaque is a mobile tool where you can share and find stories from Mozambique. From thoughts and culture, through history and gastronomy, to tourism and even fashion, if it is something about Mozambique, you should share or find it in Reaque.
```

**Review by thatfiredev — COMMENTED** [review]
@horaciocome1 that is great!
I left a few suggestions where I fixed minor typos and suggested replacing some words.

If you think this suggestion drifts away from the original one, feel free to keep the original and it will be merged as is.

File: README.md (hunk 1 above)
Comment (horaciocome1) [reply README.md]: It looks fancier. Lets keep it like this

**Comment by thatfiredev** [comment]
@horaciocome1 Perfect! Thanks

**Pull request merged**